app/assets/stylesheets/base/form.scss

```diff
@@ -0,0 +1,76 @@
+.form-speaker {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  width: 550px;
+  background-color: #FBFBFB;
+  padding: 40px;
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: space-between;
+  flex-direction: column;
+  flex-wrap: wrap;
+  text-align: center;
+  color: $primaryBackground;
+  margin: 40px;
+  font-size: 20px;
+  font-family: 'Lato', sans-serif;
+  box-shadow: 0.5px 0.5px 0.5px 0.5px #888888;
```

Comment:
if all values are `0.5px` you can use just one or less than that? and what about change this `#888888`  to `#888`